src/state/extension.ts

```diff
@@ -398,15 +398,15 @@ export class Extension implements vscode.Disposable {
    * Runs the given async function, displaying an error message and returning
    * the specified value if it throws an exception during its execution.
    */
-  public async runPromiseSafely<T>(
-    f: () => Thenable<T>,
+  public runPromiseSafely<T>(
+    f: () => T,
     errorValue: () => T,
     errorMessage: (error: any) => string,
   ) {
     this.dismissErrorMessage();
 
     try {
-      return await f();
+      return f();
```

Comment:
Thanks for looking into this! Even if it does work this is not desirable; we want to `await` `f` here, otherwise we won't catch potential errors.